Sample code in CUDA EP does not compile

**Describe the bug**
This code does not compile: https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#cc

```
/home/azureuser/cloudfiles/code/Users/nakersha/onnxruntime-inference-examples/c_cxx/api/main.cpp:32:58: error: ‘EXHAUSTIVE’ is not a member of ‘OrtCudnnConvAlgoSearch’
 options.cudnn_conv_algo_search = OrtCudnnConvAlgoSearch::EXHAUSTIVE;
                                                          ^~~~~~~~~~
/home/azureuser/cloudfiles/code/Users/nakersha/onnxruntime-inference-examples/c_cxx/api/main.cpp:35:1: error: ‘SessionOptionsAppendExecutionProvider_CUDA’ was not declared in this scope
 SessionOptionsAppendExecutionProvider_CUDA(session_options, &options);
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.11
- Python version: N/A
- Visual Studio version (if applicable): N/A
- GCC/Compiler version (if compiling from source): 7.5
- CUDA/cuDNN version: 11.4
- GPU model and memory: V100

**To Reproduce**
- Add code to a C++ application

**Expected behavior**
Code should compile

**Additional context**
Add any other context about the problem here. If the issue is about a particular model, please share the model details as well to facilitate debugging.

--- a/docs/execution-providers/CUDA-ExecutionProvider.md
+++ b/docs/execution-providers/CUDA-ExecutionProvider.md
@@ -134,7 +134,7 @@ OrtCUDAProviderOptions options;
 options.device_id = 0;
 options.arena_extend_strategy = 0;
 options.gpu_mem_limit = 2 * 1024 * 1024 * 1024;
-options.cudnn_conv_algo_search = OrtCudnnConvAlgoSearch::EXHAUSTIVE;
+options.cudnn_conv_algo_search = OrtCudnnConvAlgoSearchExhaustive;
 options.do_copy_in_default_stream = 1;
 
 SessionOptionsAppendExecutionProvider_CUDA(session_options, &options);

